docs/source/java/install.rst

```diff
@@ -101,6 +101,34 @@ transitive dependencies of Flight.
         </build>
     </project>
 
+For users that consume Arrow Java through JSE17/JSE18, please remember to
+add modules needed. As an example if you are testing Java Arrow dependencies
+and seeing error like `module java.base does not opens java.nio` please
+consider to add ``--add-opens=java.base/java.nio=ALL-UNNAMED``.
+
+Plugin configuration: Just for your test execution created.
+
+.. code-block::
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>3.0.0-M6</version>
+                <configuration>
+                        <argLine>--add-opens=java.base/java.nio=ALL-UNNAMED</argLine>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+Environment variables: To execute your Arrow Java main code.
+
+.. code-block::
+
+    _JAVA_OPTIONS="--add-opens=java.base/java.nio=ALL-UNNAMED" mvn exec:java -Dexec.mainClass="YourMainCode"
```

Comment:
Can this be configured in `pom.xml` somehow?

Reply:
Uhmm, let me figure out if there are some plugins available, but, initially I could say no way to do that.

Reply:
Hi @lidavidm, I didn't find a plugin to configure environment variable as needed on the pom.xml directly.

Reply:
There is a [exec-maven-plugin](https://www.mojohaus.org/exec-maven-plugin/) but not able to pass argLine needed

Reply:
Ok, sounds good